echopype/calibrate/calibrate_ek.py

```diff
@@ -130,22 +130,27 @@ class CalibrateEK60(CalibrateEK):
     def __init__(self, echodata, env_params, cal_params, **kwargs):
         super().__init__(echodata, env_params, cal_params)
 
-        # Set sonar_type
+        # Set sonar_type and waveform/encode mode
         self.sonar_type = "EK60"
-
-        # Get env_params
-        self.env_params = get_env_params_EK60(echodata=echodata, user_env_dict=self.env_params)
         self.waveform_mode = "CW"
         self.encode_mode = "power"
 
-        # Compute range
-        self.compute_echo_range()
-
         # Get the right ed_group for CW power samples
         self.ed_group = retrieve_correct_beam_group(
```

Comment:
Consider renaming `ed_group` to, say, `ed_beam_group`? Just for clarity.

Reply:
Sure, this is something inherited, but I agree to change it for clarify.

Reply:
Done! In the process I discovered a redundancy in input argument for `_cal_complex_samples` and `_cal_power_samples` and removed them as well.